src/main/java/org/commonjava/util/sidecar/model/TrackedContentEntry.java

```diff
@@ -0,0 +1,390 @@
+package org.commonjava.util.sidecar.model;
+
+import java.io.Externalizable;
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.commonjava.util.sidecar.model.pkg.PackageTypeConstants.PKG_TYPE_GENERIC_HTTP;
+import static org.commonjava.util.sidecar.model.pkg.PackageTypeConstants.PKG_TYPE_MAVEN;
+
+public class TrackedContentEntry
+        implements Comparable<TrackedContentEntry>,Externalizable
+{
+    private static final long serialVersionUID = 6469004486206600578L;
+
+    private static final int VERSION = 3;
+
+    private TrackingKey trackingKey;
+
+    private StoreKey storeKey;
+
+    private String accessChannel;
+
+    private String path;
+
+    private String originUrl;
+
```

Comment:
Which json do you mean? The one in archive service tarball? If so I think it is a bug.
We don't know if it is important yet because not sure how PNC is using it, but better to add it back.